include/pg_shard.h

```diff
@@ -77,6 +77,9 @@ typedef struct DistributedPlan
 } DistributedPlan;
 
 
+#define INVALID_SHARD_ID (-1)
+typedef int64 ShardId;
```

Comment:
It seems to be good idea to use separate type for shardId rahter then use int64 in many places...